demos/simple/errors/verify.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+cargo clean
+
+# this should detect multiple errors
+( cargo-verify --tests --verbose > out || true )
```

Comment:
You can use `| tee out` instead of `> out` and `cat out`.

Reply:
good idea - done